src/frontends/jax/src/op/iota.cpp

```diff
@@ -0,0 +1,37 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#include "openvino/frontend/jax/node_context.hpp"
+#include "openvino/op/constant.hpp"
+#include "openvino/op/range.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace jax {
+namespace op {
+using namespace ov::op;
+
+OutputVector translate_iota(const NodeContext& context) {
+    num_inputs_check(context, 1, 1);
+
+    // Get the dtype and size from the input
+    auto dtype = context.get_input(0);
```

Comment:
I think this is a sort of attribute like `size` so try to get it using `context.const_named_param`. You should adopt this method to retrieve `element::Type`